transformations/url_decode.go

```diff
@@ -8,18 +8,20 @@ import (
 )
 
 func urlDecode(data string) (string, error) {
-	res, _, _ := doURLDecode(data)
-	// TODO add error?
-	return res, nil
+	for i := 0; i < len(data); i++ {
+		if data[i] == '%' || data[i] == '+' {
+			// TODO add error?
```

Comment:
Is this TODO comment still relevant? I see that `doURLDecode` is not going to return any more `invalidCount`, that could have been the way to implement the error detection. 
It may be reasonable to:
- Consider the TODO no anymore relevant and just remove it (this would align it to `inplaceUniDecode`) 
or
- Keep a return value that could still be used to implement this check (even just a bool flag?)